pkgs/tools/graphics/ldgallery/compiler/generate.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p cabal2nix
+
+# Note for maintainers: keep version in sync with the ldgallery viewer
+TAG="v2.0"
```

Comment:
You could extract the version from the `default.nix` itself, like this: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/X11/opentabletdriver/update.sh#L11

Reply:
I moved the TAG out as a parameter of the script and added some asserts to ensure that all versions are in sync.